code/game/objects/structures/wild.dm

```diff
@@ -924,14 +924,14 @@ var/list/seed_list_jungle
 	name = "chinchona"
 	desc = "you can extract quinine from it."
 	icon = 'icons/obj/flora/plants.dmi'
-	icon_state = "chinchona1"
+	//icon_state = "chinchona1"
```

Comment:
Why did you remove the chinchona icon state?

Reply:
Because it doesn't exist and never has

Reply:
fixed